### ISSUE TITLE
Lack of Documentation

### ISSUE THREAD

**Issue by vaishnavm217** [pjreddie/darknet#1399]
Hi,

The darknet framework is very easy and simple to use. But `BOiiiiii`, not a single line of documentation explaining the cfg files, which is the gateway to all API. Although it can be figured out by reading the cfg files properly and relating with other frameworks, this lack of documentation makes newcomers hesistate to use this framework. It would be awesome have proper website explaining everything, rather than being like nvidia and saying `it just works`(jk)

TLDR: Proper documentation should be made atleast for cfgs and APIs.
